integrationExamples/top-level-paapi/gam-contextual.html

```diff
@@ -0,0 +1,134 @@
+<html xmlns="http://www.w3.org/1999/html">
+<head>
+    <script src="shared/example-setup.js"></script>
+    <script src="../../build/dev/prebid.js"></script>
+
+    <script async src="https://securepubads.g.doubleclick.net/tag/js/gpt.js"></script>
+
+    <script>
+        var FAILSAFE_TIMEOUT = 3300;
+        var PREBID_TIMEOUT = 3000;
+        var adUnits = [{
+                code: 'div-1',
+                mediaTypes: {
+                    banner: {
+                        sizes: [[300, 250]],
+                    }
+                },
+                ortb2Imp: {
+                    ext: {
+                        ae: 1
+                    }
+                },
+                bids: [
+                    {
+                        bidder: 'optable',
+                        params: {
+                            site: 'daa30ba1-5613-4a2c-b7f0-34e2c033202a'
+                        },
+                    },
+                    {
+                        bidder: 'contextual',
+                        params: {
+                            site: 'daa30ba1-5613-4a2c-b7f0-34e2c033202a'
+                        }
+                    }
+                ],
+            }
+            ]
+        ;
+
+        var pbjs = pbjs || {};
+        pbjs.que = pbjs.que || [];
+
+        var googletag = googletag || {};
+        googletag.cmd = googletag.cmd || [];
+        googletag.cmd.push(function () {
+            googletag.pubads().disableInitialLoad();
+        });
+
+        pbjs.que.push(function () {
+            pbjs.setConfig({
+                debug: true,
+                paapi: {
+                    enabled: true,
+                    gpt: {
+                        autoconfig: false
+                    },
+                    topLevelSeller: {
+                        auctionConfig: {
+                            seller: window.location.origin,
+                            decisionLogicURL: new URL('shared/decisionLogic.js', window.location).toString(),
+                        },
+                        overrideWinner: true
```

Comment:
So this would mean, if pbjs has a tls pa winner, go with that instead of a possble gam contextual winner?  right?

Reply:
Yes - depending on what you mean by GAM contextual winner. It means that when Prebid is asked to render a bid, it will instead render a PAAPI winning bid if it has one for the slot. 

In the typical GAM flow (at least typical in our examples, including this one) the bid that is potentially replaced is the Prebid bid that won the Prebid auction, and then the GAM auction, to be replaced by the winner of the PAAPI auction.